package.json

```diff
@@ -66,6 +66,6 @@
     "ed-html": "node reffy.js run ed html",
     "ed-diff": "node reffy.js run ed diff",
     "ed-diffnew": "node reffy.js run ed diffnew",
-    "test": "jasmine && mocha --recursive tests/"
+    "test": "npm run build && jasmine && mocha --recursive tests/"
```

Comment:
It's convenient to add it here, but I believe we still need a "test" command that does not build `browser.js`, so as to detect cases where we fail to include it in the pull request (for instance, with this change and assuming tests passed, the fact that `browser.js` wasn't in the initial PR #343 may have gone unnoticed).

Or we need to switch to a mode where `browser.js` is not tracked in the repo at all.

Reply:
I guess it is my overall preference not to include built files in repos, but I know this isn't a shared preference, and there may be good reasons to have it included here.

In any case, you're right that this is out of scope for this PR, so I will remove it

Reply:
I don't disagree. I think I initially included the build to be able to keep `rollup` as a `devDependencies`, and to be able to run Reffy commands right away without having to build anything first. But it's not a big deal to move `rollup` to the `dependencies` list (that won't be the largest one ;)), and we can easily detect the absence of `browser.js` in the code to report a nice error message, or even to run the build from the code.

I just want to avoid is a situation where parts of the code assume that the build is there while other parts don't :)